bd-grpc/src/lib.rs

```diff
@@ -501,9 +501,11 @@ impl<C: Connect + Clone + Send + Sync + 'static> Client<C> {
       },
     }
 
+    log::debug!("sending streaming request");
```

Comment:
did you want to leave these logs in? Not sure how useful they would be in general

Reply:
It would just for me debugging. I can remove them.